### ISSUE TITLE
Instagram password and username needed

### ISSUE THREAD

**Issue by Phantomx23** [Ha3MrX/InstaBrute#49]
Instagram password and username needed

**Comment by Miraltrial** [comment]
I need to get into password mobile number is not with me and email belong to someone else don't want to help

**Comment by Siya011** [comment]
I need to instagram password mobile number is not mine and not gmail account in my device only I got a username

**Comment by Siya011** [comment]
> Instagram password and username needed

Please help me

**Comment by Umiumair** [comment]
Hy

**Comment by Anushkumaruffs** [comment]
heath_ledger_33_ my Instagram id password change

**Comment by INeedReverseEngineering** [comment]
Can anyone help me get a strange ins account?

**Comment by nour545** [comment]
yes i can give you if you want ? and who wannt username and password instagramm talk with me

**Comment by ihmib28** [comment]
> oui je peux te le donner si tu veux ? et qui veut un nom d'utilisateur et un mot de passe instagramm parle avec moi

Can you help me find the password of an Instagram account please?

**Comment by Eneasotiri** [comment]
How do i run the code to get an insta page
